garak/generators/huggingface.py

```diff
@@ -159,45 +239,33 @@ def __init__(
         if not torch.cuda.is_available():
             message = "OptimumPipeline needs CUDA, but torch.cuda.is_available() returned False; quitting"
             logging.critical(message)
-            raise ValueError(message)
+            raise GarakException(message)
 
-        use_fp8 = False
+        self.use_fp8 = False
         if _config.loaded:
             if "use_fp8" in _config.plugins.generators.OptimumPipeline:
-                use_fp8 = True
-
-        self.generator = pipeline(
-            "text-generation",
-            model=self.name,
-            do_sample=self.do_sample,
-            device=self.device,
-            use_fp8=use_fp8,
-        )
-        self.deprefix_prompt = name in models_to_deprefix
+                self.use_fp8 = True
```

Comment:
is this distinct from dtype? can we unify these params so there isn't some subtle loss of config/api inconsistency when moving from e.g. `Pipeline` to `OptimumPipeline`?

Reply:
I can do some research on that, testing this generator directly may provide me some insight on that.

Reply:
Looks like `use_fp8` just maps the dtype via the quantizer: https://github.com/huggingface/transformers/blob/2d4820284d823ad362e63662397aae957a95901b/src/transformers/quantizers/quantizer_quanto.py#L157